app/controllers/mentorships_controller.rb

```diff
@@ -25,6 +25,7 @@ def create
   end
 
   def show
+    raise
```

Comment:
Hey Matt, I'm reviewing the code and just wondering why there's a raise put here? Can I delete it again? Thanks